src/github/workflow.ts

```diff
@@ -0,0 +1,13 @@
+import transformWorkflow from "../transforms/workflow";
+import { Context } from "probot/lib/context";
+
+export default async (context: Context, jiraClient) => {
+	const { data: jiraPayload } = await transformWorkflow(context);
```

Comment:
Same problem here, we completely discard the `data` key, hence we don't need to have it in the function either.